sdl-core/src/main/scala/io/smartdatalake/config/ConfigToolbox.scala

```diff
@@ -59,7 +59,7 @@ object ConfigToolbox {
     val name = "interactive"
     val globalConfig = GlobalConfig()
     val context = ActionPipelineContext(name, name, SDLExecutionId.executionId1, instanceRegistry, Some(LocalDateTime.now()), SmartDataLakeBuilderConfig(name, Some(name)), phase = ExecutionPhase.Exec, serializableHadoopConf = defaultHadoopConf, globalConfig = globalConfig)
-    globalConfig._sparkSession = Some(sparkSession)
+    globalConfig._sparkSession = Option(sparkSession)
```

Comment:
Why is Option better than Some?

Reply:
If sparkSession=null then Option returns None, else Some(sparkSession)
Background: getDefaultActionPipelineContext is used in different context for unit tests and interactive tools. Sometimes you want to use an existing SparkSession, sometimes you want to work without SparkSession (e.g. create it on-demand).